packages/person/src/person-search/element.ts

```diff
@@ -0,0 +1,303 @@
+ *
+ * @fires action Fires when a selection has been made on the fwc-list element
+ */
+export class PersonSearchElement
+  extends LitElement
+  implements SearchableDropdownProps, SearchableDropdownControllerHost
+{
+  // static shadowRootOptions = { ...Object.assign(LitElement.shadowRootOptions, { delegatesFocus: true }) };
+
+  /* style object css */
+  static styles = [sddStyles];
+
+  controller = new PersonSearchController(this);
+
+  /* Label passed to the fwc-text-input component */
+  @property()
+  label = '';
+
+  /* Placeholder passe to fwc-textinput */
+  @property()
+  placeholder = '';
+
+  /* Textinput variant passed to the fwc-text-input component */
+  @property()
+  variant = 'page';
+
+  /* The leading icon to display in fwc-textinput */
+  @property()
+  leadingIcon = 'search';
+
+  /* The trailing icon to display in fwc-textinput */
+  @property({ attribute: false, state: true })
+  trailingIcon = '';
+
+  /* The icon string to render in result list items on the meta slot */
+  @property()
+  meta = '';
+
+  /* value passed to textinput */
+  @property()
+  value = '';
+
+  /* The icon string to render in result list items on the graphic slot */
+  @property()
+  graphic = '';
+
+  /* The initial text in the dropdown before keyup event */
+  @property()
+  initialText = 'Start typing to search';
+
+  /* The leading icon to display in fwc-textinput */
+  @property()
+  multiple = false;
+
+  /* Label passed to the fwc-text-input component */
+  @property()
+  dropdownHeight = '250px';
+
+  @property()
+  disabled = false;
+
+  @property()
+  selectedId = '';
+
+  @property()
+  autofocus = false;
+
+  @query('fwc-textinput')
+  textInputElement: TextInputElement | undefined;
+
+  @query('fwc-list')
+  listElement: ListElement | undefined;
+
+  /* Build fwc-list-items */
+  protected buildListItem(item: SearchableDropdownResultItem): HTMLTemplateResult {
+    this.controller._listItems.push(item.id);
+    const itemClasses = {
+      'list-item': true,
+      'item-selected': !!item.isSelected,
+      'item-error': !!item.isError,
+      'item-multiline': !!item.subTitle,
+      'item-avatar': (item.graphic ?? '').indexOf('http') > -1,
+    };
+
+    const renderItemText = () => {
+      /* Get icon for either meta or graphic slot */
+      const getIconSlot = (type: 'meta' | 'graphic') => {
+        if ((this[type] && this[type] !== 'check') || (item[type] && item[type] !== 'check')) {
+          const iconRef = item[type] ?? this[type];
+          const iconSlot = (): HTMLTemplateResult =>
+            type === 'graphic'
+              ? html`<fwc-person-avatar upn=${iconRef} size="small"></fwc-person-avatar>`
+              : html`<fwc-icon icon=${iconRef}></fwc-icon>`;
+
+          return html`<span class="slot-${type}" slot=${type}>${iconSlot()}</span>`;
+        }
+
+        return html``;
+      };
+
+      /* title and subtitle slots */
+      const generateTextContent = () => {
```

Comment:
see https://lit.dev/docs/templates/directives/#ifdefined

Reply:
Unsure of what u want here? ifDefined is for attributes, i would like to hide the title/subtitle markup if not set.